Hyperbolic_triangulation_2/tests/Hyperbolic_triangulation_2/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+# Created by the script cgal_create_cmake_script
+# This is the CMake script for compiling a CGAL application.
+
+
+project( Hyperbolic_triangulation_2_tests )
```

Comment:
Must be `Tests` (with a capital `t`) to make automated testing happy.

Reply:
Thank you for editing this, @MaelRL !
Both HT2 and P4HT2 follow this rule now.

Reply:
...which I just committed and pushed... 😅